kubernetes/yamls/elasticsearch/es-rc.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: v1
+kind: ReplicationController
```

Comment:
Instead of a replicationcontroller, use a deployment. 

Ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/